README.md

```diff
@@ -33,7 +35,7 @@ This package is designed with simplicity in mind. Originally created to allow ba
 this project has expanded to include support for basic styling as well!.
 
 ## Example Usage:
-(For a much more extensive example, look at example/main.dart).
+(For a much more extensive example, look at [example](https://github.com/hey24sheep/flutter_html/tree/master/example).
```

Comment:
Please link to the repo's example file instead of your own repository's

Reply:
@ryan-berger I have made the change. Lol, I didn't even check that I used my repo.